app/code/community/Sendsmaily/Sync/Helper/Data.php

```diff
@@ -80,6 +80,18 @@ public function newsletterOptInEnabled()
     );
   }
 
+  /**
+   * Check if sending welcome/goodbye emails via Magento is enabled.
+   *
+   * @return bool
+   */
+  public function magentoNewsletterEmailEnabled()
```

Comment:
Use `get` and `set` prefix when getting or setting a value, and `is` when you are doing a boolean check.

This should be `isMagentoNewsletterEmailEnabled`.

Reply:
This function was removed in b7ea4e51